packages/dashboard-frontend/src/overrides.css

```diff
@@ -23,6 +23,6 @@
   padding: 0 !important;
 }
 
-span[class*="label-required"] {
+span[class*='label-required'] {
```

Comment:
Stylelinter expects double quotes here; please revert.

Reply:
These changes are a result of running the command
```yarn format:fix``` in the root of che-dashboard.
WDYT?

Reply:
That's weird. Let me check it.